### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by bioy-casares**
I would consider swapping order of the sections on Comets and Planets.  I may be mistaken, but I think far more people enter Urbit via the planet flow than with a comet. 

Also, should we add a "Live Install Support" section with links to the various Telegram and Discord groups?

### CODE REVIEW

**Comment by bioy-casares** [comment]
Also, I'm relatively new to Github. Please excuse any breach of edit/suggestion etiquette.

**Comment by drbeefsupreme** [comment]
There's been several discussions this year inside Tlon on where comets should feature on this page and the wind has been blowing towards emphasizing comets. The logic is roughly this: comets have the lowest barrier for a new person to get on Urbit and can be seen as a way to test drive. You should make it as easy as possible for people to try out your product. While right now most people using Urbit have some degree of fluency with cryptoproperty, that will be less of the case over time and asking people to know about Ethereum before they can get on Urbit is a big ask, and putting planets first creates that barrier. Definitely getting everyone a planet is the ultimate goal, but comets have been increasingly viewed as the main on-ramp to Urbit this year, based on the people in Tlon I've talked to about this.

The ordering of topics on this page probably has a pronounced effect on user intake so the benefits of careful thought here are substantial. I'm pretty convinced that comets first is the right order right now, but am always willing to listen to opposing arguments, and growing user bases is entirely outside of my wheelhouse.

The calculus for this page will change substantially once hosting launches and this will deserve a new discussion at that time (or it can start now).

**Comment by drbeefsupreme** [comment]
> Also, I'm relatively new to Github. Please excuse any breach of edit/suggestion etiquette.

No breach has been made! Your contribution is very welcome.

File: content/using/install.md [hunk 1]
```diff
@@ -321,7 +321,7 @@ Setting up and running a star or galaxy is a little bit different from setting u
 Stars are valuable cryptographic assets. They can be bought in one of two ways.
 
 - Transacting privately with a star owner.
-- Buying a star on a marketplace like [OpenSea](https://opensea.io/collection/urbit-id)
+- Buying a star on a marketplace like [OpenSea](https://opensea.io/collection/urbit-id) or [Planet Market](https://planet.market)
```
Comment (drbeefsupreme) [inline content/using/install.md:324]: I don't see where you can buy a star on this page?

File: content/using/install.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ To start with a comet, continue below. For planet instructions, see [further bel
 
 ## Installing Urbit
 
-Regardless of your Urbit ID, you'll want the Urbit binary installed first. The Urbit binary runs on a Unix-like operating system – Ubuntu, Fedora, or macOS, for example.
+Regardless of your Urbit ID, you'll want the Urbit binary installed first. This is done via the command line. The Urbit binary runs on a Unix-like operating system – Ubuntu, Fedora, or macOS, for example.
```
Comment (drbeefsupreme) [inline content/using/install.md:27]: This is a great suggestion.

File: content/using/install.md (hunk 1 above)
Comment (bioy-casares) [reply content/using/install.md:324]: Planet Market sells stars via their OTC desk.  This is typically advertised on the @planet_market Twitter account.

File: content/using/install.md (hunk 2 above)
Comment (bioy-casares) [reply content/using/install.md:27]: Is there a good beginners manual for using CLI that we could include? One for each OS I imagine?

File: content/using/install.md (hunk 1 above)
Comment (matildepark) [reply content/using/install.md:324]: This isn't even [in the FAQ](https://planet.market/faq) though. It should perhaps be on the site somewhere?

Comment (bioy-casares) [reply content/using/install.md:324]: Good point.

File: content/using/install.md (hunk 2 above)
Comment (baudtack) [reply content/using/install.md:27]: Personally feel that should be an exercise left to the reader to find.

**Comment by baudtack** [comment]
@syzygyzer maybe we could get a second version of this pr with the changes that you liked but sans swapping comets and planets?

**Comment by baudtack** [comment]
Bump.

**Comment by jalehman** [comment]
This is now fairly out-of-date after a few rounds of updates have been made to the install guide. Going to close—feel free to create a new PR against the new one!

**Pull request closed without merging**